geomesa-gt/geomesa-gt-tools/examples/ingest/avro/ingest_avro.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# type and converter example-xml is registered in $GEOMESA_HOME/conf/application.conf
+bin/geomesa-gt ingest --param dbtype=postgis --param host=localhost --param port=5432 --param schema=public --param database=database --param user=postgres --param passwd=postgres -s example-avro -C example-avro-no-header example_no_header.avro
```

Comment:
Do we have a script that ties together these examples and tries 'em out?

Reply:
we have, e.g. [this](https://github.com/locationtech/geomesa/blob/geomesa_2.11-2.3.0/geomesa-accumulo/geomesa-accumulo-tools/examples/test/geomesa-test-accumulo.sh), but I don't think that those scripts are used and/or maintained...